### PR TITLE
PTX-14146: Aetos Log Refactoring to remove clutter and repeated logs

### PR DESCRIPTION
**Pull request by CharudathGopal**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by aghodke1312 — APPROVED** [review]

File: pkg/log/log.go [hunk 1]
```diff
@@ -162,8 +210,69 @@ func (mf *MyFormatter) Format(entry *logrus.Entry) ([]byte, error) {
...
+	b.WriteString(fmt.Sprintf("%s:[%s %s:#%d]  %s\n",
+		entry.Time.Format("2006-01-02 15:04:05 -0700"), level, fileName, entry.Caller.Line,
 		entry.Message))
 	return b.Bytes(), nil
 }
+
+func Fatalf(format string, args ...interface{}) {
+	dash.Fatal(format, args...)
+	tpLog.Fatalf(format, args...)
+}
```
Comment (pureneelesh) [inline pkg/log/log.go:222]: Just wondering if we could have done this by adding us as a hook under logrus. Perhaps, InfoD could have been implemented by using WithFields()?

File: pkg/restutil/restutil.go [hunk 2]
```diff
@@ -73,17 +70,17 @@ func validateURL(url string) error {
...
-	log = logInstance.GetLogInstance()
+
 	var err error
 	err = validateURL(url)
 	if err != nil {
 		return nil, 0, err
 	}
 
-	log.Tracef("%s: %s", httpMethod, url)
+	logrus.Debugf("%s: %s", httpMethod, url)
```
Comment (pureneelesh) [inline pkg/restutil/restutil.go:80]: Maybe add some comments about why we use logrus directly in this file.

File: drivers/node/gke/gke.go [hunk 3]
```diff
@@ -1,15 +1,13 @@
...
-	"github.com/sirupsen/logrus"
-
 	"github.com/libopenstorage/cloudops"
 	"github.com/libopenstorage/cloudops/gce"
 	"github.com/portworx/torpedo/drivers/node"
 	"github.com/portworx/torpedo/drivers/node/ssh"
+	"github.com/portworx/torpedo/pkg/log"
+	"os"
+	"time"
 )
```
[the remark this answers is not in the document]
Comment (CharudathGopal) [reply drivers/node/gke/gke.go:12]: I tried changing manually upon, saving its again revert to above.

File: drivers/node/aws/aws.go [hunk 4]
```diff
@@ -2,18 +2,17 @@ package aws
...
 	"github.com/aws/aws-sdk-go/service/ec2"
 	"github.com/aws/aws-sdk-go/service/ssm"
 	"github.com/portworx/sched-ops/task"
 	"github.com/portworx/torpedo/drivers/node"
-	"github.com/sirupsen/logrus"
+	"github.com/portworx/torpedo/pkg/log"
+	"os"
+	"strings"
+	"time"
 )
```
[the remark this answers is not in the document]
Comment (CharudathGopal) [reply drivers/node/aws/aws.go:16]: same here, manually edited. Upon saving reverting my change.. I am using golang BTW.

File: drivers/scheduler/k8s/k8s.go [hunk 5]
```diff
@@ -261,7 +260,6 @@ func (k *K8s) Init(schedOpts scheduler.InitOptions) error {
 	k.PureVolumes = schedOpts.PureVolumes
 	k.PureSANType = schedOpts.PureSANType
 	k.RunCSISnapshotAndRestoreManyTest = schedOpts.RunCSISnapshotAndRestoreManyTest
-	k.log = schedOpts.Logger
```
[the remark this answers is not in the document]
Comment (CharudathGopal) [reply drivers/scheduler/k8s/k8s.go:264]: This code was introduced as part of the Aetos integration, so reverted to its original change and updated the references

File: pkg/restutil/restutil.go (hunk 2 above)
Comment (CharudathGopal) [reply pkg/restutil/restutil.go:80]: This is done to avoid cyclic dependency, added comment.

**Pull request merged**